bot/discord/commands/stats.js

```diff
@@ -401,6 +401,40 @@ exports.run = async (client, message, args) => {
             .setTitle("Server stats - Error")
             .setDescription("Couldn't load the server stats.\n```\n" + error + "```");
           msg.edit({ embed: errorMsg });
+   })*/
+
+  // New start from 02.2023
+  const firstEmbed = new Discord.MessageEmbed()
```

Comment:
This would be just MessageEmbed:

![image](https://user-images.githubusercontent.com/76603072/216443457-86abb7c7-8439-42cc-8cc8-94611f2265ad.png)